src/Microsoft.AspNetCore.Http.Abstractions/HttpMethods.cs

```diff
@@ -55,5 +55,10 @@ public static bool IsTraceMethod(string method)
         {
             return object.ReferenceEquals(Trace, method) || StringComparer.OrdinalIgnoreCase.Equals(Trace, method);
         }
+
+        public static bool Equals(string left, string right)
```

Comment:
This method is pointless... We hard code a few methods there's no point having a case insensitive overload here. It adds nothing..